### PR TITLE
Rename "Support Team" to "Technical Support Team"

### PR DESCRIPTION
**Pull request by Hiviexd**
was changed today in https://osu.ppy.sh/groups/22

### CODE REVIEW

File: wiki/People/Technical_Support_Team/en.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Technical Support Team
+
+*For a team that administers accounts, see: [Account support team](/wiki/People/Account_support_team)*
+
+The **Technical Support Team**, also known as **Support Team Redux**, are members of the osu! staff whom are focused at forum moderation on several subforums: [Development](https://osu.ppy.sh/community/forums/2), [Gameplay & Rankings](https://osu.ppy.sh/community/forums/13) (excluding [Tournaments](https://osu.ppy.sh/community/forums/55) and [Mapping Techniques](https://osu.ppy.sh/community/forums/61)), [Skinning](https://osu.ppy.sh/community/forums/15), [Feature Requests](https://osu.ppy.sh/community/forums/4), and [Help](https://osu.ppy.sh/community/forums/5).
```
Comment (venix12) [inline wiki/People/Technical_Support_Team/en.md:5]:
```suggestion
The **Technical Support Team**, also known as **Support Team** or **Support Team Redux**, are members of the osu! staff whom are focused at forum moderation on several subforums: [Development](https://osu.ppy.sh/community/forums/2), [Gameplay & Rankings](https://osu.ppy.sh/community/forums/13) (excluding [Tournaments](https://osu.ppy.sh/community/forums/55) and [Mapping Techniques](https://osu.ppy.sh/community/forums/61)), [Skinning](https://osu.ppy.sh/community/forums/15), [Feature Requests](https://osu.ppy.sh/community/forums/4), and [Help](https://osu.ppy.sh/community/forums/5).
```

**Review by venix12 — CHANGES_REQUESTED** [review]
Could probably just update lists in the language articles rather than outdating them.

File: wiki/People/Technical_Support_Team/en.md (hunk 1 above)
Comment (Walavouchey) [reply wiki/People/Technical_Support_Team/en.md:5]: i believe this is more accurate

```suggestion
The **Technical Support Team**  (formerly Support Team, also known as Support Team Redux), are osu! staff members focused on forum moderation on several subforums: [Development](https://osu.ppy.sh/community/forums/2), [Gameplay & Rankings](https://osu.ppy.sh/community/forums/13) (excluding [Tournaments](https://osu.ppy.sh/community/forums/55) and [Mapping Techniques](https://osu.ppy.sh/community/forums/61)), [Skinning](https://osu.ppy.sh/community/forums/15), [Feature Requests](https://osu.ppy.sh/community/forums/4), and [Help](https://osu.ppy.sh/community/forums/5).
```

File: wiki/People/Technical_Support_Team/en.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+## Roles and responsibilities
+
+The Technical Support Team is responsible for:
+
+1. Testing and/or confirming bug reports to let the developers know and for them to resolve.
+2. Organising threads at [Feature Requests](https://osu.ppy.sh/community/forums/4) and [Help](https://osu.ppy.sh/community/forums/5), marking them as *Resolved*, *Confirmed*, *Invalid*, *Duplicate* and/or *Added*.
+
+If someone needs any assistance on the said subforums, members of this team are available to help.
```
Comment (Walavouchey) [inline wiki/People/Technical_Support_Team/en.md:14]:
```suggestion
If someone needs any assistance in the aforementioned subforums, members of this team are available to help.
```

File: wiki/People/Technical_Support_Team/en.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+## Team members
+
+The [Technical Support Team group page](https://osu.ppy.sh/groups/22) lists all of the team members.
+
+*Note: All Technical Support Team members speak English in addition to the language(s) listed below unless noted otherwise.*
```
Comment (Walavouchey) [inline wiki/People/Technical_Support_Team/en.md:20]: this note should be right below the heading, so these two paragraphs should be swapped

**Review by Walavouchey — CHANGES_REQUESTED** [review]

File: wiki/People/Technical_Support_Team/en.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# Technical Support Team
+
+*For a team that administers accounts, see: [Account support team](/wiki/People/Account_support_team)*
```
Comment (Walavouchey) [inline wiki/People/Technical_Support_Team/en.md:3]:
```suggestion
*Not to be confused with [Account support team](/wiki/People/Account_support_team).*
```

**Comment by Hiviexd** [comment]
> Could probably just update lists in the language articles rather than outdating them.

idk, there's tons of languages there that i'm not confident with, even with translation tools

**Comment by venix12** [comment]
i mean, modifying members list just boils down to removing 4 lines without adding anything

**Comment by Walavouchey** [comment]
idk how ci isn't catching it but this needs a redirect to not break links

File: wiki/redirect.yaml
Comment (agatemosu) [inline wiki/redirect.yaml:1]: I think that these lines should be moved (I don't know how the redirect works)
https://github.com/ppy/osu-wiki/blob/35e7dea93bce239da083bb63287ff385fb550d35/wiki/redirect.yaml#L593-L596
https://github.com/ppy/osu-wiki/blob/35e7dea93bce239da083bb63287ff385fb550d35/wiki/redirect.yaml#L2087

File: wiki/People/Technical_Support_Team/en.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+# Technical Support Team
+
+*Not to be confused with [Account support team](/wiki/People/Account_support_team).*
+
+The **Technical Support Team**  (formerly *Support Team*, also known as *Support Team Redux*) is a group of osu! staff members focused on forum moderation on several subforums: [Development](https://osu.ppy.sh/community/forums/2), [Gameplay & Rankings](https://osu.ppy.sh/community/forums/13) (excluding [Tournaments](https://osu.ppy.sh/community/forums/55) and [Mapping Techniques](https://osu.ppy.sh/community/forums/61)), [Skinning](https://osu.ppy.sh/community/forums/15), [Feature Requests](https://osu.ppy.sh/community/forums/4), and [Help](https://osu.ppy.sh/community/forums/5).
```
Comment (agatemosu) [inline wiki/People/Technical_Support_Team/en.md:5]:
```suggestion
The **Technical Support Team** (formerly *Support Team*, also known as *Support Team Redux*) is a group of osu! staff members focused on forum moderation on several subforums: [Development](https://osu.ppy.sh/community/forums/2), [Gameplay & Rankings](https://osu.ppy.sh/community/forums/13) (excluding [Tournaments](https://osu.ppy.sh/community/forums/55) and [Mapping Techniques](https://osu.ppy.sh/community/forums/61)), [Skinning](https://osu.ppy.sh/community/forums/15), [Feature Requests](https://osu.ppy.sh/community/forums/4), and [Help](https://osu.ppy.sh/community/forums/5).
```

File: wiki/redirect.yaml
Comment (Hiviexd) [reply wiki/redirect.yaml:1]: would it be sufficient to update the redirect on each one of those?

Comment (agatemosu) [reply wiki/redirect.yaml:1]: I mean moving the redirects to the bottom and updating them

Comment (Walavouchey) [reply wiki/redirect.yaml:1]: crash course on updating redirects:

- add redirect for the moved or renamed article
- update all redirects that point to the old article
- keys (left) are all lowercase, values (right) are case-sensitive
- keys don't support `#section-links`, but values do

you can move them to the bottom if you want

**Comment by agatemosu** [comment]
https://github.com/ppy/osu-wiki/blob/faccea893cc4c935e241c54c0c89a7cf1f119f69/meta/group-info/en.yaml#L10
should be updated?

**Comment by RockRoller01** [comment]
Probably should also update the forum list on the wiki article, as its now limited to help and FR according to the group listing

**Comment by Sitoria** [comment]
https://github.com/ppy/osu-wiki/blob/master/wiki/People/Account_support_team/en.md
seems to change first line too

> *For a team that moderates certain subforums, see: [Technical Support Team](/wiki/People/Technical_Support_Team)*

**Review by Walavouchey — DISMISSED** [review]

**Review by cl8n — APPROVED** [review]

**Pull request merged**